### PR TITLE
Lytics - Integration

### PR DESCRIPTION
**Pull request by ruchiramoitra**

### CODE REVIEW

File: v0/destinations/lytics/data/LYTICSPageScreenConfig.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "destKey": "_e",
```
Comment (humanchimp) [inline v0/destinations/lytics/data/LYTICSPageScreenConfig.json:3]: Please review [this comment about the usage of the special key `"_e"` for custom events](https://github.com/rudderlabs/rudder-sdk-js/pull/144#discussion_r527897323)

Comment (ruchiramoitra) [reply v0/destinations/lytics/data/LYTICSPageScreenConfig.json]: Done

File: v0/destinations/lytics/data/LYTICSTrackConfig.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "destKey": "_e",
```
Comment (arnab-p) [inline v0/destinations/lytics/data/LYTICSTrackConfig.json:3]: @ruchiramoitra According to [this](https://github.com/rudderlabs/rudder-sdk-js/pull/144#discussion_r527897323) do you think the `destKey` should be `event`?

File: v0/destinations/lytics/transform.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  removeUndefinedAndNullValues,
+  defaultRequestConfig,
+  flattenJson
+} = require("../../util");
+function responseBuilderSimple(message, category, destination) {
+  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
+  const response = defaultRequestConfig();
+  const { stream, apiKey } = destination.Config;
+
+  const timestamp = message.originalTimestamp;
```
Comment (arnab-p) [inline v0/destinations/lytics/transform.js:15]:
```suggestion
  const timestamp = message.originalTimestamp || message.timestamp;
```

File: v0/destinations/lytics/transform.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  response.endpoint = `${ENDPOINT}/${stream}?timestamp_field=${timestamp}`;
+  response.method = defaultPostRequestConfig.requestMethod;
+
+  response.userId = message.anonymousId;
```
Comment (arnab-p) [inline v0/destinations/lytics/transform.js:19]: Not needed. Please remove

**Review by arnab-p — CHANGES_REQUESTED** [review]
Please format the config JSON files and add the test cases.

File: v0/destinations/lytics/data/LYTICSTrackConfig.json (hunk 2 above)
Comment (ruchiramoitra) [reply v0/destinations/lytics/data/LYTICSTrackConfig.json:3]: not for track... that is the case for page views only.

File: v0/destinations/lytics/transform.js (hunk 4 above)
Comment (ruchiramoitra) [reply v0/destinations/lytics/transform.js]: removed

File: v0/destinations/lytics/transform.js (hunk 3 above)
Comment (ruchiramoitra) [reply v0/destinations/lytics/transform.js]: done

**Review by arnab-p — APPROVED** [review]

**Pull request merged**